src/wp-includes/embed.php

```diff
@@ -1244,3 +1244,23 @@ function wp_filter_pre_oembed_result( $result, $url, $args ) {
 
 	return $result;
 }
+
+/**
+ * Apply the loading="lazy" attribute to the oEmbed iframe.
```

Comment:
```suggestion
 * Applies the loading="lazy" attribute to the oEmbed iframe.
```
- Using third-person singular verbs to begin docblock descriptions.

Reply:
thanks!